ingest/validation/validate_metadata.py

```diff
@@ -808,6 +822,60 @@ def validate_collected_ontology_data(metadata, convention):
     return
 
 
+def validate_organ_region_metadata(ontology_info, region_ontology, metadata):
+    """
+    Validation for nonEBI-OLS ontology, Mouse Brain Atlas
+    """
```

Comment:
More [PEP8-conventional](https://www.python.org/dev/peps/pep-0008/#documentation-strings):

```suggestion
    """Validation for non-EBI-OLS ontology, Mouse Brain Atlas
    """
```

And similarly elsewhere.

Reply:
`bf48ed8`